scripts/pr-convergence.py

```
@@ -0,0 +1,971 @@
+def parse_repo(repo: str) -> tuple[str, str]:
+    parts = repo.split("/", 1)
+    if len(parts) != 2 or not parts[0] or not parts[1]:
+        raise ValueError("repo must use owner/name format")
+    return parts[0], parts[1]
+
+
+def sorted_dict_list(values: list[dict[str, Any]], key: str) -> list[dict[str, Any]]:
+    return sorted(values, key=lambda item: str(item.get(key, "")))
+
+
+def gh_args_are_read_only(args: list[str]) -> bool:
```

Ruff (0.16.0)

[warning] Too many return statements (8 > 6)

(PLR0911)